Shopping List/Controllers/ShoppingController.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  ShoppingController.swift
+//  Shopping List
+//
+//  Created by Angel Buenrostro on 1/18/19.
+//  Copyright © 2019 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+class ShoppingController {
+    
+    var itemsAdded = 0
+    
+    init(){
+        
+        //TODO : Try to load from UserDefaults first
+//        if UserDefaults.standard.array(forKey: "shoppingKey") == nil {
+        loadShoppingItemsFromAssets()
+//        } else { loadShoppingFromDefaults() }
+    }
+    
+    let itemNames = ["apple", "grapes", "milk", "muffin", "popcorn", "soda", "strawberries"]
+    
+    var shoppingList : [ShoppingItem] = []
+    
+    
+    func loadShoppingItemsFromAssets(){
+        
+        for item in itemNames {
+            let imageName = item
+            guard let image = UIImage(named: imageName) else { return }
+            guard let imageData = image.pngData() else { return }
+            let shoppingItem = ShoppingItem(imageData: imageData, name: imageName, addedToList: false)
+            shoppingList.append(shoppingItem)
+        }
+    }
+    
+    func loadShoppingFromDefaults(){
+        guard let persistedList = UserDefaults.standard.array(forKey: "shoppingKey") else { return }
+        for item in persistedList {
```

Comment:
To use `UserDefaults` first you need to set it, give it a value. Then check that value and implement some logic. If you go and check `Planets` project, you will see it has a simple API. Or check this [article](https://www.hackingwithswift.com/read/12/2/reading-and-writing-basics-userdefaults)